python_modules/dagster/dagster/components/testing.py

```diff
@@ -324,98 +434,74 @@ def get_component_defs_from_defs_path(
 
 
 @contextmanager
-def scaffold_defs_sandbox(
+def temp_components_sandbox(
     *,
-    component_cls: type,
-    component_path: Optional[Union[Path, str]] = None,
-    scaffold_params: Optional[dict[str, Any]] = None,
-    scaffold_format: ScaffoldFormatOptions = "yaml",
     project_name: Optional[str] = None,
-) -> Iterator[DefsPathSandbox]:
-    """Create a lightweight sandbox to scaffold and instantiate a component. Useful
+) -> Iterator[DefsSandbox]:
```

Comment:
Standardize on terms? `temp_components_sandbox` -> `DefsSandbox` is confusing

Reply:
Updated, now is `create_defs_folder_sandbox` -> `DefsFolderSandbox`.